Sources/Core/Extensions/Foundation/String+PovioKit.swift

```diff
@@ -83,7 +83,8 @@ public extension String {
   @available(iOS 15, *)
   func toMarkdown() -> AttributedString {
     do {
-      return try AttributedString(markdown: self)
+      let options = AttributedString.MarkdownParsingOptions(interpretedSyntax: .full)
+      return try AttributedString(markdown: self, options: options)
```

Comment:
Have you tested it out on how rendering improves with this change?

Reply:
Yes, I've tested it on a project where we use markdown elements. It now works with new lines, bulleted/numbered list, code block, ..., which was previously ignored.